conf/application.conf

```
@@ -117,8 +117,8 @@ feature {
 }
 
 timeout {
-  period = 900
-  countDown = 120
+  period = 20
+  countDown = 10
```

Should not be changed
```suggestion
  period = 900
  countDown = 120
```